packages/api-client/src/components/CommandPalette/TheCommandPalette.vue

```diff
@@ -153,6 +161,21 @@ const openCommandPalette = ({
   nextTick(() => commandInputRef.value?.focus())
 }
 
+/** Focus on first result when conducting a search */
+watch(commandQuery, (newQuery) => {
+  if (newQuery && searchResultsWithPlaceholderResults.value.length > 0) {
+    selectedSearchResult.value = 0
+    nextTick(() => {
```

Comment:
nit but we could do async await on next time to keep the indentation down :) but non blocking for this PR

```
watch(commandQuery, async (newQuery) => {
  if (newQuery && searchResultsWithPlaceholderResults.value.length > 0) {
    selectedSearchResult.value = 0
    await nextTick()

```